checkstyle.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN" "https://checkstyle.org/dtds/configuration_1_3.dtd">
+
+<module name="Checker">
+    <property name="fileExtensions" value="java"/>
+    <module name="NewlineAtEndOfFile"/>
+
+    <module name="TreeWalker">
+        <module name="SuppressionCommentFilter"/>
+
+        <module name="AvoidStarImport">
+            <property name="allowStaticMemberImports" value="true"/>
+            <property name="excludes" value="net.earthcomputer.clientcommands.command"/>
```

Comment:
What's this line for?

Reply:
It is to exclude the wildcard import in `ClientCommands.java` from the check.

Reply:
Okay then maybe suppress that one with a comment rather than globally in the config

Reply:
Done.